config_template.cfg

```diff
@@ -779,6 +773,17 @@ THERMAL_DIFFUSIVITY= 1.0
 
 % ---------------- ADJOINT-FLOW NUMERICAL METHOD DEFINITION -------------------%
 %
+% Frozen the slope limiter in the discrete adjoint formulation (NO, YES)
+FROZEN_LIMITER_DISC= NO
+%
+% Frozen the turbulent viscosity in the discrete adjoint formulation (NO, YES)
+FROZEN_VISC_DISC= NO
+%
+% Use an inconsistent spatial integration (primal-dual) in the discrete
+% adjoint formulation. The AD will use the numerical methods in
+% the ADJOINT-FLOW NUMERICAL METHOD DEFINITION section (NO, YES)
+INCONSITENT_DISC= NO
```

Comment:
I have never tried to use a different numerical method for the discrete adjoint. Will do that soon :+1:  Thanks for adding the other options to the config template.

BTW: There is a typo in the name of this option.